dev/packages/alpha/mysql/0.0.2/manifest.yml

```diff
@@ -8,7 +8,7 @@ type: integration
 categories:
 - logs
 - metrics
-release: ga
+release: beta
```

Comment:
I nice feature of having these as beta and the automatically imported packages as experimental is that currently the experimental ones don't show up in Kibana.